templates/static-files.conf

```diff
@@ -0,0 +1,60 @@
+server {
+        listen 80;
+        server_name {{ site_url }};
+
+        {% if use_ssl %}
+        return 301 https://{{ site_url }}$request_uri;
+        {% else %}
+        root {{ site_path }};
+
+        location / {
+                index index.html;
+        }
+
+        include /etc/nginx/locations.d/{{ site_url }}/*.conf;
+        {% endif %}
+}
+
+{% if use_ssl %}
+server {
+        listen 443 ssl http2;
+        listen [::]:443 ssl http2;
```

Comment:
How does this behave if the sysctl for IPv6 is off?  That is set on a non-trivial number of void systems at the moment.

Reply:
If you have disabled ipv6 via, i.e. `net.ipv6.conf.all.disable_ipv6` you can still attempt to bind to the ipv6 address, but none of your interfaces will actually have ipv6 addresses.

In the end, nginx doesn't care, requests to listen to ::, and keeps going

Reply:
Ok, sounds good